src/dg_wldcmd.cpp

```diff
@@ -635,6 +635,47 @@ WCMD(do_wrent) {
     remove_player_from_game(ch, QUIT_WRENT);
 }
 
+/* allow a mob to set ANY skill or spell based on targets class
```

Comment:
it's really the world not the mob here, but yeah.

Reply:
Oops, let me fix that